core/src/block.rs

```diff
@@ -188,14 +182,13 @@ mod pending {
 
             BlockBuilder(Chained(BlockPayload {
                 header: Self::make_header(
```

Comment:
Just a note.

I don't have a good feeling about silenced *dirty* (not pure) function here. I mean that it takes current time. Can we specify that in `chain()` documentation? Although it's not so related to this PR.

Reply:
done